components/dialog/dialog-fullscreen.js

```diff
@@ -226,6 +220,9 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 				: 0;
 			const startTop = mediaQueryList.matches ? 42 : 0;
 			topOverride = iframeTop + startTop;
+		} else if (window.innerWidth <= 615 || (window.innerWidth <= 900 && window.innerHeight <= 420)) {
+			heightOverride.height = `${window.innerHeight - 42 - 2}px`;
```

Comment:
Nit: a comment here like "render full window height - 42px padding - 2px border"  would be helpful for future us 😄